Runtime/Services/BacktraceApi.cs

```diff
@@ -202,11 +198,7 @@ public IEnumerator Send(string json, IEnumerable<string> attachments, Dictionary
               ? System.Diagnostics.Stopwatch.StartNew()
               : new System.Diagnostics.Stopwatch();
 
-            var requestUrl = queryAttributes != null
-                ? GetParametrizedQuery(_serverUrl.ToString(), queryAttributes)
-                : ServerUrl;
-
-            using (var request = _httpClient.Post(requestUrl, json, attachments))
+            using (var request = _httpClient.Post(ServerUrl, json, attachments, queryAttributes))
```

Comment:
query attributes are collected outside algorithm responsible for generating JSON data - for example to collect performance statistics: how much time we spend on generating report/data, storing data on the database etc - something that we can't just put in attributes. They suppose to in query attributes, but since I adjusted our API for minidump purpose they can be renamed to something else. Since you know the purpose of those attributes belong to current request or they collect statistics for current request - `requestAttributes` is a better name? What do you think?

Reply:
I like `requestAttributes`!

Reply:
@konraddysput what about `httpRequestAttributes` or `backtraceRequestAttributes` or `metaAttributes` or something?

Reply:
I just renamed it to `attributes`. Thanks for suggestion